cmd/subnetcmd/upgradecmd/vm.go

```diff
@@ -57,7 +58,7 @@ func newUpgradeVMCmd() *cobra.Command {
 
 	cmd.Flags().BoolVar(&useLatest, "latest", false, "upgrade to latest version")
 	cmd.Flags().StringVar(&targetVersion, "version", "", "Upgrade to custom version")
-	cmd.Flags().StringVar(&useBinary, "binary", "", "Upgrade to custom binary")
+	cmd.Flags().StringVar(&newBinary, "binary", "", "Upgrade to custom binary")
```

Comment:
Nit: Maybe adding the clarification that this is a file path somewhere could be a good idea

Reply:
renamed